src/java.desktop/unix/native/common/awt/fontpath.c

```diff
@@ -48,6 +48,9 @@
 #define AWT_UNLOCK()
 #endif /* !HEADLESS */
 
+/* for dlopen */
+#include <macro_helper.h>
+
```

Comment:
I have no idea why we would want to use some VM internal in the desktop module.
Please take the desktop module changes out of this PR.